From d04b3d198ab1b78b77d456c251e15a27bf905d85 Mon Sep 17 00:00:00 2001
From: hayaksi1 <193020925+hayaksi1@users.noreply.github.com>
Date: Tue, 1 Sep 2026 22:31:14 +0300
Subject: [PATCH] Refresh the two jest snapshots carrying the EventTileView
 module hash

The appearance and layout-switcher previews render real event tiles, so
their snapshots embed the CSS-module hash of the shared EventTileView
stylesheet. The fork's full-size-threads resolution edits that stylesheet,
which moves the hash; the snapshots are regenerated with the runner rather
than hand-edited.
---
 .../LayoutSwitcher-test.tsx.snap              | 32 +++++++++----------
 .../AppearanceUserSettingsTab-test.tsx.snap   | 32 +++++++++----------
 2 files changed, 32 insertions(+), 32 deletions(-)

diff --git a/apps/web/test/unit-tests/components/views/settings/__snapshots__/LayoutSwitcher-test.tsx.snap b/apps/web/test/unit-tests/components/views/settings/__snapshots__/LayoutSwitcher-test.tsx.snap
index 5506cf87e6..0277722e49 100644
--- a/apps/web/test/unit-tests/components/views/settings/__snapshots__/LayoutSwitcher-test.tsx.snap
+++ b/apps/web/test/unit-tests/components/views/settings/__snapshots__/LayoutSwitcher-test.tsx.snap
@@ -62,14 +62,14 @@ exports[`<LayoutSwitcher /> should render 1`] = `
               <div
                 aria-atomic="true"
                 aria-live="off"
-                class="_root_18x1m_9 mx_EventTile _stateOwnEvent_18x1m_903 _layoutGroup_18x1m_194"
+                class="_root_umc2n_9 mx_EventTile _stateOwnEvent_umc2n_903 _layoutGroup_umc2n_194"
                 data-event-id="$9999999999999999999999999999999999999999999"
                 data-scroll-tokens="$9999999999999999999999999999999999999999999"
                 data-testid="event-tile"
                 tabindex="-1"
               >
                 <div
-                  class="_slotSender_18x1m_104"
+                  class="_slotSender_umc2n_104"
                   data-testid="event-tile-slot-sender"
                 >
                   <div
@@ -88,7 +88,7 @@ exports[`<LayoutSwitcher /> should render 1`] = `
                   </div>
                 </div>
                 <div
-                  class="_slotAvatar_18x1m_95"
+                  class="_slotAvatar_umc2n_95"
                   data-testid="event-tile-slot-avatar"
                 >
                   <span
@@ -104,12 +104,12 @@ exports[`<LayoutSwitcher /> should render 1`] = `
                   </span>
                 </div>
                 <div
-                  class="_line_18x1m_71 mx_EventTile_line"
+                  class="_line_umc2n_71 mx_EventTile_line"
                   data-testid="event-tile-line"
                   id="1"
                 >
                   <div
-                    class="_slotBody_18x1m_96"
+                    class="_slotBody_umc2n_96"
                     data-testid="event-tile-slot-body"
                   >
                     <div
@@ -170,14 +170,14 @@ exports[`<LayoutSwitcher /> should render 1`] = `
               <div
                 aria-atomic="true"
                 aria-live="off"
-                class="_root_18x1m_9 mx_EventTile _stateOwnEvent_18x1m_903 _layoutBubble_18x1m_705"
+                class="_root_umc2n_9 mx_EventTile _stateOwnEvent_umc2n_903 _layoutBubble_umc2n_705"
                 data-event-id="$9999999999999999999999999999999999999999999"
                 data-scroll-tokens="$9999999999999999999999999999999999999999999"
                 data-testid="event-tile"
                 tabindex="-1"
               >
                 <div
-                  class="_slotSender_18x1m_104"
+                  class="_slotSender_umc2n_104"
                   data-testid="event-tile-slot-sender"
                 >
                   <div
@@ -196,7 +196,7 @@ exports[`<LayoutSwitcher /> should render 1`] = `
                   </div>
                 </div>
                 <div
-                  class="_slotAvatar_18x1m_95"
+                  class="_slotAvatar_umc2n_95"
                   data-testid="event-tile-slot-avatar"
                 >
                   <span
@@ -212,12 +212,12 @@ exports[`<LayoutSwitcher /> should render 1`] = `
                   </span>
                 </div>
                 <div
-                  class="_line_18x1m_71 mx_EventTile_line"
+                  class="_line_umc2n_71 mx_EventTile_line"
                   data-testid="event-tile-line"
                   id="2"
                 >
                   <div
-                    class="_slotBody_18x1m_96"
+                    class="_slotBody_umc2n_96"
                     data-testid="event-tile-slot-body"
                   >
                     <div
@@ -278,14 +278,14 @@ exports[`<LayoutSwitcher /> should render 1`] = `
               <div
                 aria-atomic="true"
                 aria-live="off"
-                class="_root_18x1m_9 mx_EventTile _stateOwnEvent_18x1m_903 _layoutIrc_18x1m_471"
+                class="_root_umc2n_9 mx_EventTile _stateOwnEvent_umc2n_903 _layoutIrc_umc2n_471"
                 data-event-id="$9999999999999999999999999999999999999999999"
                 data-scroll-tokens="$9999999999999999999999999999999999999999999"
                 data-testid="event-tile"
                 tabindex="-1"
               >
                 <div
-                  class="_slotTimestamp_18x1m_106"
+                  class="_slotTimestamp_umc2n_106"
                   data-testid="event-tile-slot-timestamp"
                 >
                   <span
@@ -293,7 +293,7 @@ exports[`<LayoutSwitcher /> should render 1`] = `
                   />
                 </div>
                 <div
-                  class="_slotAvatar_18x1m_95"
+                  class="_slotAvatar_umc2n_95"
                   data-testid="event-tile-slot-avatar"
                 >
                   <span
@@ -309,7 +309,7 @@ exports[`<LayoutSwitcher /> should render 1`] = `
                   </span>
                 </div>
                 <div
-                  class="_slotSender_18x1m_104"
+                  class="_slotSender_umc2n_104"
                   data-testid="event-tile-slot-sender"
                 >
                   <div
@@ -328,12 +328,12 @@ exports[`<LayoutSwitcher /> should render 1`] = `
                   </div>
                 </div>
                 <div
-                  class="_line_18x1m_71 mx_EventTile_line"
+                  class="_line_umc2n_71 mx_EventTile_line"
                   data-testid="event-tile-line"
                   id="3"
                 >
                   <div
-                    class="_slotBody_18x1m_96"
+                    class="_slotBody_umc2n_96"
                     data-testid="event-tile-slot-body"
                   >
                     <div
diff --git a/apps/web/test/unit-tests/components/views/settings/tabs/user/__snapshots__/AppearanceUserSettingsTab-test.tsx.snap b/apps/web/test/unit-tests/components/views/settings/tabs/user/__snapshots__/AppearanceUserSettingsTab-test.tsx.snap
index 202ebd98f9..f22a6c43a0 100644
--- a/apps/web/test/unit-tests/components/views/settings/tabs/user/__snapshots__/AppearanceUserSettingsTab-test.tsx.snap
+++ b/apps/web/test/unit-tests/components/views/settings/tabs/user/__snapshots__/AppearanceUserSettingsTab-test.tsx.snap
@@ -205,14 +205,14 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                       <div
                         aria-atomic="true"
                         aria-live="off"
-                        class="_root_18x1m_9 mx_EventTile _stateOwnEvent_18x1m_903 _layoutGroup_18x1m_194"
+                        class="_root_umc2n_9 mx_EventTile _stateOwnEvent_umc2n_903 _layoutGroup_umc2n_194"
                         data-event-id="$9999999999999999999999999999999999999999999"
                         data-scroll-tokens="$9999999999999999999999999999999999999999999"
                         data-testid="event-tile"
                         tabindex="-1"
                       >
                         <div
-                          class="_slotSender_18x1m_104"
+                          class="_slotSender_umc2n_104"
                           data-testid="event-tile-slot-sender"
                         >
                           <div
@@ -231,7 +231,7 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                           </div>
                         </div>
                         <div
-                          class="_slotAvatar_18x1m_95"
+                          class="_slotAvatar_umc2n_95"
                           data-testid="event-tile-slot-avatar"
                         >
                           <span
@@ -247,12 +247,12 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                           </span>
                         </div>
                         <div
-                          class="_line_18x1m_71 mx_EventTile_line"
+                          class="_line_umc2n_71 mx_EventTile_line"
                           data-testid="event-tile-line"
                           id="1"
                         >
                           <div
-                            class="_slotBody_18x1m_96"
+                            class="_slotBody_umc2n_96"
                             data-testid="event-tile-slot-body"
                           >
                             <div
@@ -313,14 +313,14 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                       <div
                         aria-atomic="true"
                         aria-live="off"
-                        class="_root_18x1m_9 mx_EventTile _stateOwnEvent_18x1m_903 _layoutBubble_18x1m_705"
+                        class="_root_umc2n_9 mx_EventTile _stateOwnEvent_umc2n_903 _layoutBubble_umc2n_705"
                         data-event-id="$9999999999999999999999999999999999999999999"
                         data-scroll-tokens="$9999999999999999999999999999999999999999999"
                         data-testid="event-tile"
                         tabindex="-1"
                       >
                         <div
-                          class="_slotSender_18x1m_104"
+                          class="_slotSender_umc2n_104"
                           data-testid="event-tile-slot-sender"
                         >
                           <div
@@ -339,7 +339,7 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                           </div>
                         </div>
                         <div
-                          class="_slotAvatar_18x1m_95"
+                          class="_slotAvatar_umc2n_95"
                           data-testid="event-tile-slot-avatar"
                         >
                           <span
@@ -355,12 +355,12 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                           </span>
                         </div>
                         <div
-                          class="_line_18x1m_71 mx_EventTile_line"
+                          class="_line_umc2n_71 mx_EventTile_line"
                           data-testid="event-tile-line"
                           id="2"
                         >
                           <div
-                            class="_slotBody_18x1m_96"
+                            class="_slotBody_umc2n_96"
                             data-testid="event-tile-slot-body"
                           >
                             <div
@@ -421,14 +421,14 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                       <div
                         aria-atomic="true"
                         aria-live="off"
-                        class="_root_18x1m_9 mx_EventTile _stateOwnEvent_18x1m_903 _layoutIrc_18x1m_471"
+                        class="_root_umc2n_9 mx_EventTile _stateOwnEvent_umc2n_903 _layoutIrc_umc2n_471"
                         data-event-id="$9999999999999999999999999999999999999999999"
                         data-scroll-tokens="$9999999999999999999999999999999999999999999"
                         data-testid="event-tile"
                         tabindex="-1"
                       >
                         <div
-                          class="_slotTimestamp_18x1m_106"
+                          class="_slotTimestamp_umc2n_106"
                           data-testid="event-tile-slot-timestamp"
                         >
                           <span
@@ -436,7 +436,7 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                           />
                         </div>
                         <div
-                          class="_slotAvatar_18x1m_95"
+                          class="_slotAvatar_umc2n_95"
                           data-testid="event-tile-slot-avatar"
                         >
                           <span
@@ -452,7 +452,7 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                           </span>
                         </div>
                         <div
-                          class="_slotSender_18x1m_104"
+                          class="_slotSender_umc2n_104"
                           data-testid="event-tile-slot-sender"
                         >
                           <div
@@ -471,12 +471,12 @@ exports[`AppearanceUserSettingsTab should render 1`] = `
                           </div>
                         </div>
                         <div
-                          class="_line_18x1m_71 mx_EventTile_line"
+                          class="_line_umc2n_71 mx_EventTile_line"
                           data-testid="event-tile-line"
                           id="3"
                         >
                           <div
-                            class="_slotBody_18x1m_96"
+                            class="_slotBody_umc2n_96"
                             data-testid="event-tile-slot-body"
                           >
                             <div
-- 
2.52.0

